packages/button/button.scss

```diff
@@ -40,17 +40,25 @@ $button-primary-background-color--inverted: $hvit;
     --button-text-color: #{$button-text-color};
     --button-background-color: #{$button-background-color};
     --button-focus-color: #{$button-focus-color};
+    --button-action-focus-color: #{$info--darkbg};
     --button-primary-text-color: #{$button-primary-text-color};
     --button-primary-background-color: #{$button-primary-background-color};
+    --button-action-background-color: #{$info};
+    --button-action-text-color: #{$hvit};
+    --button-action-focus-text-color: #{$svart};
 }
 
 [data-theme="dark"] {
     --button-border-color: #{$button-border-color--inverted};
     --button-text-color: #{$button-text-color--inverted};
     --button-background-color: #{$button-background-color--inverted};
     --button-focus-color: #{$button-focus-color--inverted};
+    --button-action-focus-color: #{$info--darkbg};
     --button-primary-text-color: #{$button-primary-text-color--inverted};
     --button-primary-background-color: #{$button-primary-background-color--inverted};
+    --button-action-background-color: #{$info--darkbg};
+    --button-action-text-color: #{$svart};
+    --button-action-focus-text-color: #{$svart};
```

Comment:
Mangler inverted farger for call to action knapp? Fikk den ikke til å fungere i eksemplet i hvert fall 🤔